### DISCUSSION TITLE
Chunck render distance

### DISCUSSION THREAD

**Discussion by Wegord0** [PrismarineJS/mineflayer#3085] [Q&A]
can i lower the chunk render distance of the bot to the minimum?
and if is a yes, how?

**Comment by SonicandTailsCD** [comment]
Hm...
I'll check the API and I'll be right back :)

**Comment by SonicandTailsCD** [comment]
Yes, you can! Here we go:
https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#mineflayercreatebotoptions:~:text=Create%20and%20return%20an%20instance%20of%20the%20class%20bot.
https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#botsettingsviewdistance
Hope this helps :)
